### PR TITLE
Advent 2016: Added post and assets for Dec 28th

### PR DESCRIPTION
**Pull request by Xercoy**

### CODE REVIEW

File: content/advent-2016/promoting-the-quality-and-collaboration-of-your-open-source-project.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
++++
+author = ["Corey Prak"]
+date = "2016-12-28T00:00:00-08:00"
+title = "Promoting the Quality and Collaboration of Your Open Source Project"
+series = ["Advent 2016"]
+draft = true
++++
+
+So your open source project is on Github. It has tests, an awesome logo, probably a few stars, and maybe even a few other contributors. To spread awareness, it might be shared on the relevant subreddit, Twitter, Hacker News, etc.
```
Comment (bradleyfalzon) [inline content/advent-2016/promoting-the-quality-and-collaboration-of-your-open-source-project.md]: I believe `Github` should be spelt `GitHub`.

File: content/advent-2016/promoting-the-quality-and-collaboration-of-your-open-source-project.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+While exposure is one of the most effective ways to promote a project, there are various steps that can be taken to ensure that its growth is positive and that the community it revolves around thrives.
+
+This posts includes suggestions which promote code confidence and collaboration. They are:
```
Comment (bradleyfalzon) [inline content/advent-2016/promoting-the-quality-and-collaboration-of-your-open-source-project.md]: I think `posts` should be `post`.

File: content/advent-2016/promoting-the-quality-and-collaboration-of-your-open-source-project.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+
+In addition to autocompletion of the community name and link, one benefit is an option to automagically create a PR to update the README with a badge. Sadly, this didn't work for me in the multiple communities I've created.
+
+Invites to join your community can be sent by email or from a list of Github followers. After the community is created, an invite can be sent via other platforms such as Facebook, Twitter, LinkedIn, and Google+.
+
+**Manually adding a Gitter Badge to a README**
+
+![](https://img.shields.io/gitter/room/nwjs/nw.js.svg)
+
+[Shields](https://shields.io) provides a slick Gitter badge (repo [here](https://github.com/badges/shields)). Make the necessary substitution below, then update a README to include this badge which directs users to any desired Gitter community:
```
Comment (bradleyfalzon) [inline content/advent-2016/promoting-the-quality-and-collaboration-of-your-open-source-project.md]: Should `then update a README` be `then update the README`?

File: content/advent-2016/promoting-the-quality-and-collaboration-of-your-open-source-project.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+- [vscode-go](https://gitter.im/Microsoft/vscode-go)
+- [gvt](https://gitter.im/FiloSottile/gvt)
+- [hugo](https://gitter.im/spf13/hugo)
+- [gocode](https://gitter.im/nsf/gocode)
+
+# 2. Go Report Card
+
+![](/postimages/advent-2016/promoting-the-quality-and-collaboration-of-your-open-source-project/report_card_cover_page.png)
+
+[Go Report Card](http://goreportcard.com) is a product of the 2015 [Gopher Gala](http://gophergala.com/blog/gopher/gala/2015/02/03/winners/) hackathon. Created by [Shaun Smith](https://github.com/shawnps) and [Herman Schaaf](https://github.com/hermanschaaf) of [gojp](https://github.com/gojp), the Go Report Card is a convenient and powerful tool which validates an open source repo in multiple aspects, from running Go specific tools against the repo such as [go vet](https://golang.org/cmd/vet/), [go lint](https://github.com/golang/lint), and [ineffassign](https://github.com/gordonklaus/ineffassign) to important but.
```
Comment (bradleyfalzon) [inline content/advent-2016/promoting-the-quality-and-collaboration-of-your-open-source-project.md]: This sentence seems to have trailed off ` and ineffassign to important but.`

File: content/advent-2016/promoting-the-quality-and-collaboration-of-your-open-source-project.md [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+Creating a Report Card page for a repo is straightforward. One method is to type in the `go get` path on the Go Report Card homepage. Another method is to navigate to the following link with the go get path substituted (the first method does the same thing):
+```
+https://goreportcard.com/report/<YOUR GO GET PATH HERE>
+```
+
+**Adding a Badge**
+
+![Go Reporcard Badge](https://goreportcard.com/badge/github.com/xercoy/blobs)
+
+Make the necessary substitution to the following Markdown content which adds a badge, then click on the link to be redirected to the repo's Report Card page:
```
Comment (bradleyfalzon) [inline content/advent-2016/promoting-the-quality-and-collaboration-of-your-open-source-project.md]: I think `Markdown` can be all lowercase here.

File: content/advent-2016/promoting-the-quality-and-collaboration-of-your-open-source-project.md [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+[![Go Report Card](https://goreportcard.com/badge/<YOUR GO GET PATH HERE>](https://goreportcard.com/report/<YOUR GO GET PATH HERE>)
+```
+
+---
+
+# 3. GoDoc
+
+![](/postimages/advent-2016/promoting-the-quality-and-collaboration-of-your-open-source-project/godoc_cover_page.png)
+
+[GoDoc](https://godoc.org/-/about) hosts and generates documentation from Go source code on Bitbucket, Github, Google Project Hosting, and Launchpad. This resource makes unfamiliar code much more digestible, a detail that is paramount to anyone who is using a package or just trying one out. The effectiveness of the page generated by Godoc is similar to the package documentation from [golang.org](https://golang.org/pkg/fmt).
```
Comment (bradleyfalzon) [inline content/advent-2016/promoting-the-quality-and-collaboration-of-your-open-source-project.md]: `Godoc` should be `GoDoc` in the second instance, like in the first.

**Comment by Xercoy** [comment]
@bradleyfalzon thanks so much for the review, I'm making the corrections now and will be reviewing the post in its entirety a few more times.

Two smaller concerns that I've thought of is the title of the blog post being too long and the images of the blog post being too large.

**Pull request merged**